stock_move_force_reservation/models/stock_move.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Akretion France (http://www.akretion.com/)
+# @author: Mathieu Delva <mathieu.delva@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    show_force_quantity = fields.Boolean(compute="_compute_show_force_quantity")
+    qty_in_stock = fields.Float(compute="_compute_qty_in_stock")
```

Comment:
IMHO, those two fields either have too generic names for this module, either should go in another one as it seems not related directly to 'force reservation' flow.